src/vec.cpp

```diff
@@ -899,10 +905,16 @@ vec grid_volume::dz() const {
 }
 
 grid_volume volone(double zsize, double a) {
+  if (!_wholenumber(zsize) || !_wholenumber(a))
```

Comment:
It's not clear to me that we need `a` to be an integer.

Reply:
Isn't the criterion that `zsize * a` is an integer?

Reply:
It looks like GitHub may have lost a comment I made here earlier, or perhaps I just didn't click the button before closing the tab...

I ran some experiments with `vol2d` where just `a` was a floating point value, and while there was still some variance in the volumes, they were very minor. So it looks like the warning may not be needed there, or at least it is less important. I went ahead and made the change to remove the check of the `a` parameter, but it's in a separate commit so it will be easy to revert if we decide otherwise.

Or were you just commenting about just the `volone` case?

Reply:
Ah, I found my earlier comment in the other change discussion. The PR was probably updated before I started typing and I didn't notice that I was in the wrong field.